README.md

```diff
@@ -73,6 +73,33 @@ maintainable as possible against the Decidim core. Try to avoid hard core
 customizations which require lots of efforts to maintain over Decidim's core
 updates.
 
+## Equity data
+
+One of the main changes we made from the City of Helsinki site (other than the
+look and feel) is pulling in quantitative equity data from the City of 
+Seattle's Racial and Social Equity Index.
+
+http://data-seattlecitygis.opendata.arcgis.com/datasets/SeattleCityGIS::racial-and-social-equity-composite-index
+
```

Comment:
```suggestion
look and feel) is pulling in quantitative equity data from the [City of 
Seattle's Racial and Social Equity Index](http://data-seattlecitygis.opendata.arcgis.com/datasets/SeattleCityGIS::racial-and-social-equity-composite-index
).

```
We could have this as a link, but we don't have to

Reply:
Fixed